transport/http/retry.go

```diff
@@ -0,0 +1,5 @@
+package http
+
+type Retryer interface {
+	// TODO: Fill this interface
+}
```

Comment:
This change seems out of place, or is missing corresponding changes.

Reply:
Needs discussion